README.md

```diff
@@ -83,6 +83,26 @@ The following changes are now available in production:
 
 ### individuals-charges-api
 
+New Endpoint: `Delete Pension Charges` - Tax Year Specific 
+
+Updated Endpoint: `Amend Pension Charges`, see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#22nd-february-2023).
```

Comment:
I think this URL is incorrect as it links to the wiki, but this changelog is in the README